app/javascript/src/controller_branches.js

```diff
@@ -119,7 +124,7 @@ BranchesController.addBranchMenu = function(args) {
   var first = $(".Branch", $form).get(0) == branch.$node.get(0);
   if(!first) {
     new ActivatedMenu($ul, {
-      activator_text: "Activator text here",
+      activator_text: "Edit branch",
```

Comment:
does the JS have access to the `I18n` to make use of the translations?

Reply:
Good point.  I'eve updated this to use the locales text.